build/pkgs/gap/spkg-check.in

```diff
@@ -10,7 +10,8 @@ cd pkg/io
 make
 cd ../..
 
-make testinstall
+# This is the same as 'dev/ci.sh testinstall' (but dev/ci.sh is not part of the GAP tarball)
+gap tst/testinstall.g
```

Comment:
Maybe use the GAP you just built to perform the tests, not one that is already installed
```suggestion
./gap tst/testinstall.g
```

Reply:
Thanks @fingolfin, I'll try this